app/validation/validators.py

```diff
@@ -235,3 +232,39 @@ def __call__(self, form, from_field, to_field):
 
         if from_date == to_date or from_date > to_date:
             raise validators.ValidationError(self.messages['INVALID_DATE_RANGE'])
+
+
+class SumCheck(object):
+    def __init__(self, messages=None, currency=None):
+        if not messages:
+            messages = error_messages
+        self.messages = messages
```

Comment:
we've still not got a way to use bespoke error messages as messages can not be set.

Reply:
@dwyeradam Have added it to https://trello.com/c/YDXDfWm7/1927-refactor-of-validatorspy-and-questionnaireformpy as AC. Or if we really need it for now i can put it in to questionnaire form to pass it on to validators

Reply:
All our other validation allows for bespoke messages while this currently doesn't so it needs to.

Reply:
All our other validation allows for bespoke messages while this currently doesn't so it needs to.